README.md

```diff
@@ -65,9 +66,24 @@ For an out-of-the-box setup, you can use my docker-compose recipe or a docker im
 
 For automating the analysis of your Scala projects, check out my sbt plugin [mwz/sbt-sonar](https://github.com/mwz/sbt-sonar).
 
+# Sonar-scanner properties
+The plugin exposes the following properties which can be passed to sonar-scanner when running an analysis:
+- *sonar.sources* (optional) - Scala source directory relative to the root of your project (defaults to `src/main/scala`)
+- *sonar.scala.version* (optional) - defines the version of Scala used in your project (defaults to `2.11.0`)
+- *sonar.scoverage.reportPath* (optional) - relative path to the scoverage report (defaults to `target/scala-${sonar.scala.version}/scoverage-report/scoverage.xml`).
+
+See an example usage:
+```bash
+sonar-scanner -Dsonar.projectName=test \
+              -Dsonar.projectKey=test \
+              -Dsonar.sources=src/main/scala \
+              -Dsonar.sourceEncoding=UTF-8 \
+              -Dsonar.scala.version=2.12.5 \
+              -Dsonar.scoverage.reportPath=target/scala-2.12/scoverage-report/scoverage.xml
+```
 
 # Development
-To build the project from source, run the `assembly` task in sbt shell and the jar assembled with all of the dependencies required by this plugin should be created in the `target/scala-2.12` directory. 
+To build the project from sources, run the `assembly` task in sbt shell and the jar assembled with all of the dependencies required by this plugin should be created in the `target/scala-2.12` directory. 
 
 
 # Credits
```

Comment:
IMHO, this section should be rewritten too.

Reply:
I've added a section on debugging in d593721, but I'm open to hearing your suggestions about how we should rewrite this section.

This isn't really related to the release, so I'd prefer to address this separately if that's ok with you?

Reply:
Hello @mwz, I don't know if I missclicked or what happened, but my comment was intended for the **Credits** section. And the reason why I proposed it, was that when I read it the first time, I understood that this was being developed by _Sagacify_, instead of being a continuation from a fork of that project.
You know, English is not my mother language, and sometimes I don't get the correct meaning of a sentence at first. So I would like to apologize for all this.

In any case the debugging part was very nice.

Reply:
Ok, so I've slightly tweaked the wording in the Credits section, but as you said, we probably need to think about rewriting this section as we now have a brand new Scoverage sensor and we should probably mention about this somewhere.

Let's do this in a separate PR though, if you don't mind.